src/main/resources/default.template.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
Could you move this file to a package like nl.biopet.utils? It's not best practice to have this files in the root, might overwrite files from other packages.